### DISCUSSION TITLE
ERROR - unpack requires a buffer of 271 bytes

### DISCUSSION THREAD

**Discussion by MissPuTrick** [anasty17/mirror-leech-telegram-bot#808] [Q&A]
I use auto rss 
Command: /qbleech
[qibyauto-logs-1658634365063.txt](https://github.com/anasty17/mirror-leech-telegram-bot/files/9175071/qibyauto-logs-1658634365063.txt)

**Comment by anasty17** [comment]
Mostly your problem from RSS_USER_SESSION_STRING
